File: core-java-program/leet-code-codebase/ReverseNumber.java
import java.util.*;
public class ReverseNumber{
	public static void main(String[] args){
		Scanner sc = new Scanner(System.in);
		int num = sc.nextInt();
		System.out.println("Reversed " + reverse(num));
	}
	public static int reverse(int num){
		int rev = 0;
		while(num != 0){
			int digit = num%10;
			rev = rev*10 + digit;
			num = num/10;
		}
		return rev;
	}
}

File: core-csharp-program/leet-code-codebase/ReverseNumber.cs
using System;

public class ReverseNumber
{
    public static void Main(string[] args)
    {
        Console.Write("Enter a number: ");
        int num = int.Parse(Console.ReadLine());
        Console.WriteLine("Reversed: " + Reverse(num));
    }

    public static int Reverse(int num)
    {
        int rev = 0;
        while (num != 0)
        {
            int digit = num % 10;
            rev = rev * 10 + digit;
            num = num / 10;
        }
        return rev;
    }
}
